src/plugins/GUITestBase/src/GTUtilsMdi.cpp

```diff
@@ -264,7 +264,7 @@ void GTUtilsMdi::activateWindow(HI::GUITestOpStatus &os, const QString &windowNa
     MainWindow* mw = AppContext::getMainWindow();
     GT_CHECK(mw != NULL, "MainWindow == NULL");
 
-    CHECK(!activeWindowTitle(os).contains(windowName), );
+    CHECK(!(activeWindowTitle(os) == windowName), );
```

Comment:
Be ready to fix tests, that expect the previous behavior.

Reply:
In case of too many test fails, i'll change my scenario.